### PR TITLE
worked on sql api->select page

### LINKED ISSUE #2921: Proofread and apply style guide to the SQL API->SELECT page

### PR DESCRIPTION
**Pull request by martyna-mindsdb**
Fixes #2921 

## Please describe what changes you made, in as much detail as possible
  - worked on sql api->select page
[SELECT - MindsDB Documentation.pdf](https://github.com/mindsdb/mindsdb/files/9569601/SELECT.-.MindsDB.Documentation.pdf)

mindsdb

### CODE REVIEW

File: docs/mindsdb-docs/docs/sql/api/select.md [hunk 1]
```diff
@@ -2,47 +2,93 @@
 
 ## Description
 
-The `#!sql SELECT` statement is used to get a predictions from the model table. The data is not persistent and is returned on the fly as a result-set. The basic syntax for selecting from the model is:
+The `#!sql SELECT` statement fetches predictions from the model table. The data is returned on the fly and not saved.
+
+But there are ways to save predictions data! You can save your predictions as a view using the [`CREATE VIEW`](/sql/create/view/) statement. Another way is to insert your predictions into a table using the [`INSERT INTO`](/sql/api/insert/) statement.
```
Comment (StpMax) [inline docs/mindsdb-docs/docs/sql/api/select.md:7]: `You can save your predictions as a view` - that is not saving of prediction. In `view` user may save query to make predictions.

File: docs/mindsdb-docs/docs/sql/api/select.md [hunk 2]
```diff
@@ -2,47 +2,93 @@
...
 FROM mindsdb.[predictor_name]
 WHERE [column]=[value] 
 AND [column]=[value];
 ```
 
-!!! warning "Grammar matters"
-    Ensure that there are no spaces between the column name, equal sign and value. Ensure to not use any quotations for numerical values and singular quotes for strings.
+!!! warning "Grammar Matters"
+    Here are some points to keep in mind while writing queries in MindsDB:<br/>
+    &nbsp;&nbsp;&nbsp;1. Ensure that there are no spaces between `[column]` and `=` and between `=` and `[value]`.<br/>
```
Comment (StpMax) [inline docs/mindsdb-docs/docs/sql/api/select.md:24]: That is obsolete. For me queries is working with any count of spaces anywhere.

Comment (StpMax) [reply docs/mindsdb-docs/docs/sql/api/select.md:24]: Instead of p1. we can add:
`[column]=[value]` may be joined only via `AND`

**Review by StpMax — CHANGES_REQUESTED** [review]

File: docs/mindsdb-docs/docs/sql/api/select.md (hunk 1 above)
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/api/select.md:7]: You are correct.
I have mentioned that on the `CREATE VIEW` page (PR #2909 is already merged) as follows:
> The `#!sql CREATE VIEW` statement creates a view, which is a saved `SELECT` statement executed every time we call this view, as opposed to a table that stores its data in columns and rows.

I will make the correction here.

Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/api/select.md:7]: There is a link to the `CREATE VIEW` page, and also I made this correction:
> But there are ways to save predictions data! You can save your predictions as a view using the [`CREATE VIEW`](/sql/create/view/) statement. Please note that a view is a saved query and does not store data like a table. Another way is to insert your predictions into a table using the [`INSERT INTO`](/sql/api/insert/) statement.

File: docs/mindsdb-docs/docs/sql/api/select.md (hunk 2 above)
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/api/select.md:24]: I will change point 1 to the following:
> The `[column]=[value]` pairs may be joined by `AND` or `OR` keywords.

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**